Package.swift

```diff
@@ -226,22 +227,39 @@ let package = Package(
     ),
     .testTarget(
       name: "CoreUnit",
-      dependencies: ["FirebaseCore", "SharedTestUtilities", "OCMock"],
+      dependencies: [
+        "FirebaseCore",
+        "SharedTestUtilities",
+        "HeartbeatLoggingTestUtils",
+        "OCMock",
+      ],
       path: "FirebaseCore/Tests/Unit",
       exclude: ["Resources/GoogleService-Info.plist"],
       cSettings: [
         .headerSearchPath("../../.."),
       ]
     ),
+
+    // MARK: - Heartbeat Logging
+
     .target(
       name: "HeartbeatLogging",
       path: "HeartbeatLogging/Sources/"
     ),
+    .target(
+      name: "HeartbeatLoggingTestUtils",
+      dependencies: ["HeartbeatLogging"],
+      path: "HeartbeatLoggingTestUtils/Sources/"
+    ),
     .testTarget(
       name: "HeartbeatLoggingTests",
-      dependencies: ["HeartbeatLogging"],
+      dependencies: [
+        "HeartbeatLogging",
+        "HeartbeatLoggingTestUtils",
+      ],
       path: "HeartbeatLogging/Tests/"
     ),
```

Comment:
This organization seems to be working well for SPM.

Reply:
I've been experimenting with how to make a similar setup work for CocoaPods without making FirebaseCore depend on a new, public HeartbeatLogging.podspec. 

I'm going to try following approach: 
- Create a HeartbeatLoggingTestingUtils.podspec
- Depend on new test util podspec's Swift bridging header in the FirebaseCore objc unit testing spec

Reply:
WIP with above approach at 4e9c9a9. I'm running into two compiler errors that I added comments for.

Reply:
That might work. We may need to wrap the test spec in an environment variable like https://github.com/google/GoogleDataTransport/blob/main/GoogleDataTransport.podspec#L64 so that the test is only seen by `pod lib lint` but not by `pod spec lint` or in production.